x-pack/plugins/secops/public/containers/overview/overview_network/index.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { getOr } from 'lodash/fp';
+import React from 'react';
+import { Query } from 'react-apollo';
+import { pure } from 'recompose';
+
+import { GetSiemOverviewNetworkQuery, SiemOverviewNetworkData } from '../../../graphql/types';
+import { createFilter } from '../../helpers';
+import { QueryTemplateProps } from '../../query_template';
+
+import { overviewNetworkQuery } from './index.gql_query';
+
+export interface SiemOverviewNetworkArgs {
+  id: string;
+  overviewNetwork: SiemOverviewNetworkData;
+}
+
+export interface SiemOverviewNetworkProps extends QueryTemplateProps {
+  children: (args: SiemOverviewNetworkArgs) => React.ReactNode;
+  sourceId: string;
+  endDate: number;
+  poll: number;
+  startDate: number;
+}
+
+export const SiemOverviewNetworkQuery = pure<SiemOverviewNetworkProps>(
+  ({ id = 'overviewNetworkQuery', children, filterQuery, sourceId, startDate, endDate }) => (
+    <Query<GetSiemOverviewNetworkQuery.Query, GetSiemOverviewNetworkQuery.Variables>
+      query={overviewNetworkQuery}
+      fetchPolicy="cache-and-network"
+      notifyOnNetworkStatusChange
+      variables={{
+        sourceId,
+        timerange: {
+          interval: '12h',
+          from: startDate!,
+          to: endDate!,
+        },
+        filterQuery: createFilter(filterQuery),
+      }}
+    >
+      {({ data }) => {
+        const overviewNetwork = getOr({}, `source.SiemOverviewNetwork`, data);
+        return children({
```

Comment:
Question: Should we return loading too so we can show to the end user that we are loading the data for them

Reply:
Design says to return '--' here for loading: https://elastic.slack.com/archives/C7QC1JV6F/p1554227120056100